### PR TITLE
libllvm: update from 14.0.1 to 14.0.3

### PR DESCRIPTION
**Pull request by buttaface**
Also, enable python support for lldb, as done by @thunder-coding.

### CODE REVIEW

**Comment by buttaface** [comment]
@thunder-coding, I tried upgrading the lldb package alone with the one built by this pull, because I didn't want to screw up my existing clang 14.0.1 install, and I see this output for any lldb command:
```
Traceback (most recent call last):                                       
    File "<string>", line 1, in <module>                                      
ModuleNotFoundError: No module named 'lldb'
```
Is that because the other 14.0.2 packages [install more needed files](https://stackoverflow.com/questions/30869945/how-to-import-lldb-in-a-python-script) or do you need to add something else?

**Comment by thunder-coding** [comment]
> Is that because the other 14.0.2 packages install more needed files or do you need to add something else?

Not downloaded and tested the llvm 14.0.2 debs, but I guess that's because `$prefix/lib/python3.10/site-packages/lldb` and `$prefix/lib/python3.10/six.py` should belong to the lldb subpackage.

File: packages/libllvm/build.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@ TERMUX_PKG_DESCRIPTION="Modular compiler and toolchain technologies library"
 TERMUX_PKG_LICENSE="NCSA"
```
Comment (licy183) [inline packages/libllvm/build.sh:3]: Seems that LLVM has switched its license to `Apache 2.0 with LLVM Exception` (https://releases.llvm.org/14.0.0/LICENSE.TXT). Should we change it here?

Comment (buttaface) [reply packages/libllvm/build.sh:3]: Yep, I will change it.

**Comment by buttaface** [comment]
@thunder-coding, you were right, added those Python modules to the lldb package now.

**Pull request merged**